content/teams/Tstar/index.md

```diff
@@ -0,0 +1,146 @@
+---
+type: "team"                   
+title: Tstar
+name: "Tstar"
+github-link: "https://github.com/Neelkukreti/CodeCampSubmission.git"
+description: "hello I am Bot Developer"
```

Comment:
I am pretty sure You are not a BOT right ? SO please add somethings. Maybe it should Sound something Geeky or nerdy.